doc/user_doc/vic_admin/delete_vch_options.md

```diff
@@ -43,6 +43,21 @@ Wrap the password in single quotation marks (') on Mac OS and Linux and in doubl
 
 <pre>--password '<i>esxi_host_or_vcenter_server_p@ssword</i>'</pre>
 
+### `thumbprint` ###
```

Comment:
@stuclem I recommend having a section with common options.
Target, user, password, and thumbprint are common to all vic-machine commands.
I suspect we'll end up making -force, -timeout (or similar), and -debug common across the set as well eventually.

It's going to be painful to duplicate the help for those across all the sections.